src/main/java/dk/kb/storage/facade/DsStorageFacade.java

```diff
@@ -68,6 +68,20 @@ public static void createOrUpdateRecord(DsRecordDto record)  {
         });
     }
 
+    
+    /**
+     * Update kaltura id for a record. The kaltura id is given to the record when uploaded to Kaltura. The Kaltura Id must then later be updated with this method.
+     * 
+     * @param kalturaReferenceId The referenceId given to the record when uploaded to Kaltura
+     * @param kalturaInternalId The internal Kaltura id in Kaltura. The id is given to a record after upload.
+     */
+    public static void updateKalturaInternalId(String kalturaReferenceId, String kalturaInternalId){
+         performStorageAction("updateKalturaInternalId(" + kalturaReferenceId + ")", storage -> {
+         storage.updateKalturaInternal(kalturaReferenceId, kalturaInternalId);         
+        return null;    // Something must be returned
```

Comment:
Just a question and a suggestion. Would it make sense to return 0 instead of null as an indicator of the process running without errors?